src/routes/(console)/organization-[organization]/billing/+page.ts

```diff
@@ -57,6 +59,31 @@ export const load: PageLoad = async ({ parent, depends }) => {
               organization?.billingPlan !== BillingPlan.GITHUB_EDUCATION))
         : false;
 
+    // load organization usage data and project metadata for planSummary component
+    let organizationUsage = null;
+    let usageProjects: Record<string, UsageProjectInfo> = {};
+
+    try {
+        organizationUsage = await sdk.forConsole.billing.listUsage(organization.$id);
+    } catch (e) {
+        organizationUsage = null;
+    }
+
+    try {
+        const projectsResponse = await sdk.forConsole.projects.list([
+            Query.equal('teamId', organization.$id),
+            Query.limit(1000)
+        ]);
```

Comment:
check with Damodar if we can improve this or only load when needed, this is a very expensive call. We could use pagination on UI with Skeleton loaders or similar approach. Check with design as well if needed.